esp/useful_scripts/server_setup/new_site.sh

```diff
@@ -378,40 +391,54 @@ fi
 # To reset: remove appropriate section from Apache config
 if [[ "$MODE_APACHE" || "$MODE_ALL" ]]
 then
+    # TODO(benkraft): what the heck does this DAV thing do, and do we need it?
```

Comment:
No idea. I doubt we need it. I wouldn't be surprised if it's no longer functional; I personally didn't reconfigure anything DAV-related during the most recent server reinstall.